### PR TITLE
Optimization of copy as python

### PR DESCRIPTION
**Pull request by lvybriage**
#904 copy trial parameters
delete control page code

some style change: 
1. experiment font-color
2. change nni icon
3. table title Id -> ID
4. default metric column left

### CODE REVIEW

File: src/webui/src/components/TrialsDetail.tsx [hunk 1]
```diff
@@ -38,21 +38,21 @@ class TrialsDetail extends React.Component<{}, TrialDetailState> {
     public tableList: TableList | null;
 
     private titleOfacc = (
-        <Title1 text="Default Metric" icon="3.png" />
+        <Title1 text="Default metric" icon="3.png" />
     );
 
     private titleOfhyper = (
-        <Title1 text="Hyper Parameter" icon="1.png" />
+        <Title1 text="Hyper parameter" icon="1.png" />
```
Comment (QuanluZhang) [inline src/webui/src/components/TrialsDetail.tsx]: Hyperparameter or Hyper-parameter

Comment (lvybriage) [reply src/webui/src/components/TrialsDetail.tsx]: Hyper-parameter. ok.

File: src/webui/src/components/overview/SuccessTable.tsx [hunk 2]
```diff
@@ -46,17 +46,17 @@ class SuccessTable extends React.Component<SuccessTableProps, {}> {
 
         let bgColor = '';
         const columns = [{
-            title: 'Trial No.',
+            title: 'Trial no.',
```
Comment (QuanluZhang) [inline src/webui/src/components/overview/SuccessTable.tsx]: should be No.

File: src/webui/src/components/public-child/LogPath.tsx [hunk 3]
```diff
@@ -31,13 +31,13 @@ class LogPath extends React.Component<LogpathProps, {}> {
                             />
                             <LogPathChild
                                 eachLogpath={logStr.split(',')[1]}
-                                logName="hdfsLogPath:"
+                                logName="hdfs log path:"
```
Comment (QuanluZhang) [inline src/webui/src/components/public-child/LogPath.tsx]: HDFS

File: src/webui/src/components/public-child/PaiTrialLog.tsx [hunk 4]
```diff
@@ -40,19 +40,19 @@ class PaitrialLog extends React.Component<PaitrialLogProps, {}> {
...
                                         <Row>
                                             <LogPathChild
                                                 eachLogpath={logStr.split(',')[0]}
-                                                logName="trial stdout:"
+                                                logName="Trial stdout:"
                                             />
                                             <LogPathChild
                                                 eachLogpath={logStr.split(',')[1]}
-                                                logName="hdfsLog:"
+                                                logName="hdfs log:"
```
Comment (QuanluZhang) [inline src/webui/src/components/public-child/PaiTrialLog.tsx]: Log on HDFS

File: src/webui/src/components/trial-detail/DefaultMetricPoint.tsx [hunk 5]
```diff
@@ -63,8 +63,8 @@ class DefaultPoint extends React.Component<DefaultPointProps, DefaultPointState>
                 },
                 formatter: function (data: TooltipForAccuracy) {
                     const result = '<div class="tooldetailAccuracy">' +
-                        '<div>Trial No: ' + data.data[0] + '</div>' +
-                        '<div>Default Metric: ' + data.data[1] + '</div>' +
+                        '<div>Trial no: ' + data.data[0] + '</div>' +
```
Comment (QuanluZhang) [inline src/webui/src/components/trial-detail/DefaultMetricPoint.tsx]: No.

File: src/webui/src/components/trial-detail/TableList.tsx [hunk 6]
```diff
@@ -131,13 +131,13 @@ class TableList extends React.Component<TableListProps, TableListState> {
         const wantResult: Array<string> = [];
         Object.keys(checkedValues).map(m => {
             switch (checkedValues[m]) {
-                case 'Trial No.':
-                case 'Id':
+                case 'Trial no.':
```
Comment (QuanluZhang) [inline src/webui/src/components/trial-detail/TableList.tsx]: No.

File: src/webui/src/components/trial-detail/Intermeidate.tsx [hunk 7]
```diff
@@ -109,7 +109,7 @@ class Intermediate extends React.Component<IntermediateProps, IntermediateState>
                             obj = temp.hyperPara;
                         }
                         return '<div class="tooldetailAccuracy">' +
-                            '<div>Trial Id: ' + trialId + '</div>' +
+                            '<div>Trial id: ' + trialId + '</div>' +
```
Comment (QuanluZhang) [inline src/webui/src/components/trial-detail/Intermeidate.tsx]: ID

File: src/webui/src/components/trial-detail/TableList.tsx [hunk 8]
```diff
@@ -226,9 +226,9 @@ class TableList extends React.Component<TableListProps, TableListState> {
         Object.keys(columnSelected).map(key => {
             const item = columnSelected[key];
             switch (item) {
-                case 'Trial No.':
+                case 'Trial no.':
```
Comment (QuanluZhang) [inline src/webui/src/components/trial-detail/TableList.tsx]: No.

File: src/webui/src/components/trial-detail/TableList.tsx [hunk 9]
```diff
@@ -226,9 +226,9 @@ class TableList extends React.Component<TableListProps, TableListState> {
...
                     showColumn.push({
-                        title: 'Trial No.',
+                        title: 'Trial no.',
```
Comment (QuanluZhang) [inline src/webui/src/components/trial-detail/TableList.tsx]: No.

File: src/webui/src/static/const.ts [hunk 10]
```diff
@@ -21,11 +21,11 @@ const MONACO = {
 };
 const COLUMN_INDEX = [
     {
-        name: 'Trial No.',
+        name: 'Trial no.',
```
Comment (QuanluZhang) [inline src/webui/src/static/const.ts]: No.

File: src/webui/src/components/overview/SuccessTable.tsx (hunk 2 above)
Comment (lvybriage) [reply src/webui/src/components/overview/SuccessTable.tsx]: ok

File: src/webui/src/static/const.ts [hunk 11]
```diff
@@ -45,11 +45,11 @@ const COLUMN_INDEX = [
         index: 10000
     },
     {
-        name: 'Intermediate Result',
+        name: 'Intermediate result',
         index: 10001
     }
 ];
-const COLUMN = ['Trial No.', 'Id', 'Duration', 'Status', 'Default', 'Operation', 'Intermediate Result'];
+const COLUMN = ['Trial no.', 'ID', 'Duration', 'Status', 'Default', 'Operation', 'Intermediate result'];
```
Comment (QuanluZhang) [inline src/webui/src/static/const.ts]: No.

File: src/webui/src/components/public-child/LogPath.tsx (hunk 3 above)
Comment (lvybriage) [reply src/webui/src/components/public-child/LogPath.tsx]: ok

File: src/webui/src/components/overview/Progress.tsx [hunk 12]
```diff
@@ -200,87 +202,85 @@ class Progressed extends React.Component<ProgressProps, ProgressState> {
...
                             <div>{trialNumber.runTrial}</div>
                         </Row>
                     </Col>
-                </Row>
-                <Row className="mess">
-                    <Col span={8}>
+                    <Col span={6}>
                         <Row className="basic colorOfbasic">
-                            <p>succeed</p>
+                            <p>Succeed</p>
```
Comment (SparkSnail) [inline src/webui/src/components/overview/Progress.tsx]: Succeeded?

File: src/webui/src/components/trial-detail/DefaultMetricPoint.tsx (hunk 5 above)
Comment (lvybriage) [reply src/webui/src/components/trial-detail/DefaultMetricPoint.tsx]: ok

File: src/webui/src/components/trial-detail/TableList.tsx (hunk 6 above)
Comment (lvybriage) [reply src/webui/src/components/trial-detail/TableList.tsx]: ok

File: src/webui/src/components/trial-detail/Intermeidate.tsx (hunk 7 above)
Comment (lvybriage) [reply src/webui/src/components/trial-detail/Intermeidate.tsx]: ok

File: src/webui/src/components/trial-detail/TableList.tsx (hunk 8 above)
Comment (lvybriage) [reply src/webui/src/components/trial-detail/TableList.tsx]: ok

File: src/webui/src/components/trial-detail/TableList.tsx (hunk 9 above)
Comment (lvybriage) [reply src/webui/src/components/trial-detail/TableList.tsx]: ok

File: src/webui/src/static/const.ts (hunk 10 above)
Comment (lvybriage) [reply src/webui/src/static/const.ts]: ok

File: src/webui/src/static/const.ts (hunk 11 above)
Comment (lvybriage) [reply src/webui/src/static/const.ts]: ok

File: src/webui/src/components/overview/Progress.tsx (hunk 12 above)
Comment (lvybriage) [reply src/webui/src/components/overview/Progress.tsx]: ok

Comment (QuanluZhang) [reply src/webui/src/components/overview/Progress.tsx]: yes

File: src/webui/src/components/public-child/PaiTrialLog.tsx (hunk 4 above)
Comment (lvybriage) [reply src/webui/src/components/public-child/PaiTrialLog.tsx]: ok
